README.md

```diff
@@ -152,7 +154,7 @@ This operation will execute in a multi-process manner by chunking up the patient
 | `SCALED_EXEC_MAX_JOBSIZE` | Maximum patients to put in each worker job.                                 | 15            |
 | `SCALED_EXEC_STRATEGY`    | Patient source strategy to use for scaled calculation (`mongo` or `bundle`) | bundle        |
 
-Check out the [$evaluate-measure operation spec](https://www.hl7.org/fhir/measure-operation-evaluate-measure.html) for more information.
+Check out the [$evaluate operation spec](https://www.hl7.org/fhir/measure-operation-evaluate.html) for more information.
```

Comment:
```suggestion
Check out the [$evaluate operation spec](https://build.fhir.org/ig/HL7/davinci-deqm/OperationDefinition-evaluate.html) for more information.
```
Dead link :( change to this DEQM one instead!

Reply:
Actually since we don't totally have this implemented according to DEQM yet, I would add this link: https://hl7.org/fhir/R4/measure-operation-evaluate-measure.html and then maybe say something like "It currently follows the R4 $evaluate-measure specification, but we are transitioning it to the DEQM $evaluate operation"